frontend/src/containers/Billings/BillingForm__FirstStep.tsx

```diff
@@ -0,0 +1,137 @@
+import React from "react";
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  Button,
+  FormControl,
+  FormLabel,
+  Input,
+  Select,
+} from "@chakra-ui/react";
+import { Member } from "../../utils/fetcher";
+import {
+  Event,
+  BillingForm,
+  FormMode,
+} from "../../machines/billings/billingMachine";
+
+type Props = {
+  billingForm: BillingForm;
+  formMode: FormMode;
+  members: Member[];
+  send: (event: Event) => void;
+};
+
+const BillingFormFirstStep = ({
+  members,
+  send,
+  billingForm,
+  formMode,
+}: Props) => {
+  const handleCloseForm = () => {
+    send({ type: "CANCEL_FILL_FORM" });
+  };
+
+  return (
+    <Modal isOpen={true} onClose={handleCloseForm} isCentered>
+      <ModalOverlay
+        bg="blackAlpha.300"
+        backdropFilter="blur(10px) hue-rotate(90deg)"
+      />
+      <ModalContent>
+        <ModalHeader textAlign="center">Billing Form</ModalHeader>
+        <ModalCloseButton />
+        <ModalBody pb={6}>
+          <FormControl>
+            <FormLabel>Title</FormLabel>
+            <Input
+              placeholder="Please kindly fill the billing title"
+              value={billingForm?.title}
+              onChange={(event) =>
+                send({
+                  type: "UPDATE_FORM",
+                  billingForm: {
+                    ...billingForm,
+                    title: event.target.value,
+                  },
+                })
+              }
+            />
+          </FormControl>
+
+          <FormControl mt={4}>
+            <FormLabel>Total</FormLabel>
+            <Input
+              type="number"
+              placeholder="Please kindly fill the billing total"
+              value={billingForm?.bill_amount}
+              onChange={(event) =>
+                send({
+                  type: "UPDATE_FORM",
+                  billingForm: {
+                    ...billingForm,
+                    bill_amount: event.target.valueAsNumber,
+                  },
+                })
+              }
+            />
+          </FormControl>
+
+          <FormControl mt={4}>
+            <FormLabel>Charged Member</FormLabel>
+            <Select
+              value={billingForm?.charged_member_id}
```

Comment:
ini si `billingForm` gak ada posibility undefined kan ya ? kayaknya gak perlu pake `?`  ? jadinya tinggal `billingForm.charged_member_id`

Reply:
Sorry, I think I miss to update the context in this previous state transition. Let me update it ya